QUESTIONS.md

```diff
@@ -3,3 +3,11 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+- Can we have 10-10-10 challenges around using REACT (small CRUD exercise?)
```

Comment:
Think a 10-10-10 on this stuff, maybe out functions vs. components would be great!